fearless/Modules/Staking/StakingPayoutConfirmation/StakingPayoutConfirmationInteractor.swift

```diff
@@ -109,11 +109,24 @@ final class StakingPayoutConfirmationInteractor {
             options: options
         )
     }
+
+    private func getRewardData() {
```

Comment:
Actually the logic will be similar as for account and we don't need a separate model (actually the interactor passes models to presenter, and presenter communicates with view view via view models)